requirements.txt

```diff
@@ -25,3 +25,4 @@ typing-extensions==4.5.0 ; python_full_version >= "3.7.2" and python_version < "
 ujson==5.7.0 ; python_full_version >= "3.7.2" and python_version < "3.12"
 urllib3==2.0.2 ; python_full_version >= "3.7.2" and python_version < "3.12"
 zipp==3.15.0 ; python_full_version >= "3.7.2" and python_version < "3.9"
+schema==0.7.5; python_full_version>= "2.6.0" and python_version < "3.9"
```

Comment:
With these settings as is, it was just crashing on launch for me. I'm running python 3.9.6 on this computer. 
Additionally we don't support python 2, why not make the schema from "3.7.2" and "3.12"? (We're not supporting 3.12 yet)